### PR TITLE
Create text about "explorer" in future section

### PR DESCRIPTION
**Pull request by david-a-wheeler**
This creates some basic text about the explorer in the "future" section.

We have a basic understanding what explorer is likely to look like. I thought it'd be better to give a preview, so that people will know what's likely to be coming *and* so that we can refine the text so it's (mostly) ready when the code is ready.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1879,11 +1939,162 @@ has the given correct types.
...
+It says `wff -. ph`, that is, if some `ph` is a wff
+(an expression that is true or false), then
+`-. ph` is also a wff.
+The sequence `-.` represents "logical not", and this axiom
+allows us to use the sequence `-. ph` as a wff where `ph` is a wff.
+
+Assertion 4 is a similar axiom, stating that
+`( ph -> ps )` is a well-formed formula (`wff`).
+Notice the parentheses; since they are part of the only axiom allowing
+the use of `->`, the parentheses required in this case.
```
Comment (david-a-wheeler) [inline docs/index.md:1996]: Missing "are" after "parentheses"

File: docs/index.md [hunk 2]
```diff
@@ -1879,11 +1939,162 @@ has the given correct types.
...
+Metamath proofs include proofs of the types of each expression;
+you can decide whether or not to see this.
+
+Clicking on a use of a hypothesis step id will show
+the individual assertion tab for that id (creating the tab if necessary).
+
+At the beginning of the statement is a small "+" (reveal/hide),
+where you can reveal or hide a visualization of that step.
+
+Clicking on a reference to an assertino will show an inidivual assertion tab
```
Comment (david-a-wheeler) [inline docs/index.md:2095]: The word "assertion" is misspelled here.

File: docs/index.md [hunk 3]
```diff
@@ -1857,19 +1857,79 @@ You can follow
...
+Each assertion shows its count, the type of assertion
+(theorem or axiom), and the assertion id in bold.
+The rest of the display shows a list of 0 or more hypotheses,
+each prefixed with large black circle "&#x2B24;".
+The final line of an assertion states what can be concluded
+via this assertion when its hypotheses are true.
+
 You can use the text selector to copy useful portions of any statement.
+Next to the name of each axiom or theorem is a ">" symbol which lets you
+can expand or hide its description.
```
Comment (expln) [inline docs/index.md:1905]: `which lets you can expand` -> `which lets you expand`

File: docs/index.md [hunk 4]
```diff
@@ -1879,11 +1939,162 @@ has the given correct types.
...
+Each assertion shows its count, the type of assertion
+(theorem or axiom), and the assertion id in bold.
+The rest of the display shows a list of 0 or more hypotheses,
+each prefixed with large black circle "&#x2B24;".
+The final line of an assertion states what can be concluded
+via this assertion when its hypotheses are true.
+
+You can use the text selector to copy useful portions of any statement.
+Next to the name of each axiom or theorem is a ">" symbol which lets you
+can expand or hide its description.
```
Comment (expln) [inline docs/index.md:2063]: `which lets you can expand` -> `which lets you expand`

**Review by expln — APPROVED** [review]
Great explanation!

**Pull request merged**